.github/workflows/release.yml

```diff
@@ -0,0 +1,71 @@
+name: Release CI Pipilne
+
+on:
+  release:
+    types: [ published ]
+
+
+jobs:
+  build-and-push:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout
+        uses: action/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Get Java Version
```

Comment:
It doesn't look like we do any building of the application in this container using java and maven? So we don't need to install java because the build process will happen in another docker container with a multistage build. Leaving it here won't affect anything but it takes more time to do the release.